io.github.limo_app.limo.json

```diff
@@ -0,0 +1,293 @@
+  ],
+  "cleanup":
+  [
+    "/lib/debug/bin",
+    "/lib/debug/lib",
+    "/lib/debug/source",
+    "/bin/bsd*",
+    "/bin/cbindgen",
+    "/bin/curl*",
+    "/bin/mk-ca-bundle.pl",
+    "/share/doc",
+    "/share/man",
+    "/share/pkgconfig",
+    "/include",
+    "/lib/*.a",
+    "/lib/cmake",
+    "/lib/pkgconfig",
+    "/local"
+  ],
+  "modules":
+  [
+    {
+      "name": "libarchive",
+      "buildsystem": "cmake-ninja",
+      "config-opts":
+      [
+        "-DCMAKE_BUILD_TYPE=Release",
+        "-DENABLE_TEST=OFF",
+        "-DBUILD_SHARED_LIBS=OFF"
+      ],
+      "sources":
+      [
+        {
+          "type": "git",
+          "url": "https://github.com/libarchive/libarchive.git",
+          "tag": "v3.7.4"
+        }
+      ]
+    },
+    {
+      "name": "jsoncpp",
+      "buildsystem": "cmake-ninja",
+      "builddir": true,
+      "config-opts":
+      [
+        "-DCMAKE_BUILD_TYPE=Release",
+        "-DJSONCPP_WITH_TESTS=OFF",
+        "-DJSONCPP_WITH_POST_BUILD_UNITTEST=OFF",
+        "-DBUILD_SHARED_LIBS=OFF",
+        "-DBUILD_OBJECT_LIBS=OFF"
+      ],
+      "sources":
+      [
+        {
+          "type": "git",
+          "url": "https://github.com/open-source-parsers/jsoncpp.git",
+          "tag": "1.9.5"
+        }
+      ]
+    },
+    {
+      "name": "pugixml",
+      "buildsystem": "cmake-ninja",
+      "config-opts":
+      [
+        "-DCMAKE_BUILD_TYPE=Release"
+      ],
+      "sources":
+      [
+        {
+          "type": "git",
+          "url": "https://github.com/zeux/pugixml.git",
+          "tag": "v1.14"
+        }
+      ]
+    },
+    {
+      "name": "cpr",
+      "buildsystem": "cmake-ninja",
+      "config-opts":
+      [
+        "-DCMAKE_BUILD_TYPE=Release",
+        "-DCPR_USE_SYSTEM_CURL=ON",
+        "-DBUILD_SHARED_LIBS=OFF"
+      ],
+      "sources":
+      [
+        {
+          "type": "git",
+          "url": "https://github.com/libcpr/cpr.git",
+          "tag": "1.10.5"
+        }
+      ]
+    },
+    {
+      "name": "libtbb",
+      "buildsystem": "cmake-ninja",
+      "config-opts":
+      [
+        "-DCMAKE_BUILD_TYPE=Release",
+        "-DTBB_TEST=OFF",
+        "-DBUILD_SHARED_LIBS=OFF",
+        "-DTBB_STRICT=OFF"
+      ],
+      "sources":
+      [
+        {
+          "type": "git",
+          "url": "https://github.com/oneapi-src/oneTBB.git",
+          "tag": "v2021.13.0"
+        }
+      ]
+    },
+    {
+      "name": "boost",
+      "buildsystem": "cmake-ninja",
+      "config-opts":
+      [
+        "-DCMAKE_BUILD_TYPE=Release",
+        "-DBOOST_INCLUDE_LIBRARIES=algorithm;crc;graph"
+      ],
+
+      "sources":
+      [
+        {
+          "type": "archive",
+          "url": "https://github.com/boostorg/boost/releases/download/boost-1.85.0/boost-1.85.0-cmake.tar.xz",
+          "sha256": "0a9cc56ceae46986f5f4d43fe0311d90cf6d2fa9028258a95cab49ffdacf92ad"
+        }
+      ]
+    },
+    {
+      "name": "cbindgen",
```

Comment:
Is this a build dependency only? You are cleaning up `cbindgen` which is the only thing this builds.

If so, then add `cleanup: ["*"]` here

Reply:
Yes, cbindgen is only used to build libloot